services/galley/test/integration/API/Util.hs

```diff
@@ -275,9 +278,17 @@ createBindingTeamInternalNoActivate name owner = do
   tid <- randomId
   let nt = BindingNewTeam $ newNewTeam (unsafeRange name) DefaultIcon
   _ <-
-    put (g . paths ["/i/teams", toByteString' tid] . zUser owner . zConn "conn" . zType "access" . json nt) <!! do
-      const 201 === statusCode
-      const True === isJust . getHeader "Location"
+    put
+      ( g
+          . paths ["/i/teams", toByteString' tid]
+          . zUser owner
+          . zConn "conn"
+          . zType "access"
+          . json nt
+      )
+      <!! do
+        const 201 === statusCode
+        const True === isJust . getHeader "Location"
```

Comment:
why all this reformatting as part of this PR? Makes it much harder to see actual changes that happened.

Reply:
We had some issues with ormolu, I might have missed some places.